components/zenFunctions/init_component.sqf

```diff
@@ -45,6 +45,8 @@ if !(isNil 'zen_custom_modules_fnc_register') then
     ["[CA2] Respawners", "Respawn Random Player (INDFOR)", _spawnerIndfor] call zen_custom_modules_fnc_register;
     ["[CA2] Respawners", "Respawn Random Player (Civilian)", _spawnerCiv] call zen_custom_modules_fnc_register;
 
+    // Jammer Modules
```

Comment:
i don't like the spacing in this file because i'm horrible and pedantic.  it's consistent though so shruggo